.github/workflows/linux_build.sh

```diff
@@ -4,13 +4,15 @@ set -euo pipefail
 
 df -H .
 checkout_dir=$PWD
+export YB_DOWNLOAD_EXTRACT_ONLY=${DOWNLOAD_EXTRACT_ONLY:-}
 echo "Building in directory: $checkout_dir"
 docker run -t \
   --cap-add=SYS_PTRACE \
   -e GITHUB_TOKEN \
   -e SNYK_TOKEN \
   -e YB_BUILD_THIRDPARTY_ARGS \
   -e YB_THIRDPARTY_ARCHIVE_NAME_SUFFIX \
+  -e YB_DOWNLOAD_EXTRACT_ONLY \
```

Comment:
I think this should be called "skip upload".

Reply:
Updated the variable name